src/lib.rs

```diff
@@ -546,15 +545,21 @@ impl Adapter {
     }
 
     /// Requests a connection to a physical device, creating a logical device.
+    /// Returns the device together with a queue that executres command buffers.
     ///
     /// # Panics
     ///
     /// Panics if the extensions specified by `desc` are not supported by this adapter.
-    pub fn request_device(&self, desc: &DeviceDescriptor) -> Device {
-        Device {
+    pub fn request_device(&self, desc: &DeviceDescriptor) -> (Device, Queue) {
```

Comment:
How would this work with multiple queues? We'd still want to provide the device in `request_device` but the queue would be separate.

Reply:
We'll see when we get there, perhaps? So far the working group hasn't been super active on the topic.